contributors/guide/README.md

```diff
@@ -152,7 +152,7 @@ The bot may also make some helpful suggestions for commands to run in your PR to
 Common new contributor PR issues are:
 
 * not having correctly signed the CLA ahead of your first PR (see [Sign the CLA](#sign-the-cla) section)
-* finding the right SIG or reviewer(s) for the PR (see [Code Review](#code-review) section) and following any SIG specific contributing guidelines
+* finding the right SIG or reviewer(s) for the PR (see [Code Review](#code-review) section) and following any SIG or repository specific contributing guidelines
```

Comment:
Based on discussion, I think it would be useful to have this line modification be:

```
-* finding the right SIG or reviewer(s) for the PR (see [Code Review](#code-review) section) and following any SIG specific contributing guidelines
+* finding the right SIG or reviewer(s) for the PR (see [Code Review](#code-review) section) and following any SIG or repository specific contributing guidelines (see [Learn about SIGs](#learn-about-sigs) section)
```

Reply:
I'm not sure that it's actually all that useful, since the list of SIGs doesn't really link to any CONTRIBUTING.md files.

Honestly I would be more in favor of linking to the #learn-about-sigs section since it explains about the CONTRIBUTING.md files (and not every SIG has them, either).

Reply:
Whoa. I seem to be unable to read this afternoon. I agree with Tim! @palash25, if you could make those changes, this LGTM.